bin/marathon-framework

```diff
@@ -72,7 +72,13 @@ function load_options_and_log {
     cmd+=( "$env_op" )
   done
   echo "${cmd[@]}"
-  logged marathon "${cmd[@]}" "$@"
+
+  if [[ "${cmd[@]} $@" == *'--no-logger'* ]]
+  then
+    marathon "${cmd[@]}" "$@"
```

Comment:
Ok I got it. True "marathon" in L78  is not needed, it was used for token in ``logged``,
I will create a change.